packages/experience-builder-sdk/src/blocks/DesignComponent.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+// Feel free to do any magic as regards variable definitions for design components
+// Or if this isn't necessary by the time we figure that part out, we can bid this part farewell
+export const DesignComponent = ({ ...props }) => {
+  // Using a display contents so design component content/children
+  // can appear as if they are direct children of the div wrapper's parent
+  return <div {...props} style={{ display: 'contents' }} />;
```

Comment:
According to [caniuse.com](https://caniuse.com/?search=display%20contents), this display value is not yet stable and might have some issues, e.g. with buttons.

Is the original container that was removed to a design component, a children of the design component in here?

In that case, I would suggest to directly render this container as before and not render a div-wrapper around it.

Ofc, this means we need to assume that the componentTree of a design component can only have one direct child on the first layer and this child must be a container. Every other child on top level won't be rendered (and we would have to make sure that there is only one node on root-level when updating a design component)